pages/album/[id].vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts" setup>
+const { t } = useI18n();
+toolbarTitleStore().setReactiveToolbarTitle(() => t("nav.album"));
```

Comment:
Maybe someone else will have to answer this ... Wouldn't it make sense to have the name of the album as title in the toolbar?

Reply:
Just for clarification - this sets the red, not the blue text. The blue text is fixed as of today. ![Screenshot 2023-04-27 at 10 20 36](https://user-images.githubusercontent.com/392589/234803352-dafbb221-2c16-42c1-aeac-45ed7475d6f5.png)

Reply:
I'd propose to only take the name of the album. Many websites have a breadcrumb menu, but this might not be valuable for our website, because we do not have such a nested structure of websites that this could be useful 😁

>We could do it for playlists as well.

I agree 😎